docs/getstarted.adoc

```diff
@@ -76,6 +76,11 @@ $ allure --version
 ----
 
 == Test execution
+[WARNING]
+====
+If you are using IDEA or other IDE to run test cases and build the report locally (without CI), you need to add allure.properties to your project.
```

Comment:
Suggested change

> If you are using IDE to run tests locally it may ignore Allure configuration specified in build file (as IntelliJ IDEA does). In order to make it work consider using `allure.properties` file to configure Allure. Check out <<_configuration, configuration section>> for more information.

Reply:
Updated